python/interpret_community/widget/js/main.js

```diff
@@ -1,23 +1,40 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import axios from 'axios';
 
 import { ExplanationDashboard } from 'interpret-dashboard';
 
 const RenderDashboard = (divId, data) => {
   let generatePrediction = (postData) => {
-    return fetch(data.predictionUrl, {method: "post", body: JSON.stringify(postData), headers: {
-      'Content-Type': 'application/json'
-    }}).then(resp => {
-      if (resp.status >= 200 && resp.status < 300) {
-        return resp.json()
-      }
-      return Promise.reject(new Error(resp.statusText))
-    }).then(json => {
-      if (json.error !== undefined) {
-        throw new Error(json.error)
-      }
-      return Promise.resolve(json.data)
-    })
+    if (data.withCredentials) {
+        var headers_data = {
+            'Accept': 'application/json,text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.9',
+            'Content-Type': 'application/json',
+        }
+        axios.defaults.withCredentials = true
+        var axios_options = { headers: headers_data, withCredentials: true }
+        return axios.post(data.predictionUrl, JSON.stringify(postData), axios_options)
+            .then((response) => {
+                return response.data
+            })
+            .catch(function (error) {
+                throw new Error(error)
+            })
+    } else {
+        return fetch(data.predictionUrl, {method: "post", body: JSON.stringify(postData), headers: {
+            'Content-Type': 'application/json'
+          }}).then(resp => {
+            if (resp.status >= 200 && resp.status < 300) {
+              return resp.json()
+            }
+            return Promise.reject(new Error(resp.statusText))
+          }).then(json => {
+            if (json.error !== undefined) {
+              throw new Error(json.error)
+            }
+            return Promise.resolve(json.data)
+          })
+    }
   }
 
   ReactDOM.render(<ExplanationDashboard
```

Comment:
be sure to merge with master, don't want to revert to v1 dashboard :)

Reply:
merged with master